exercise2/assets/style.css

```diff
@@ -0,0 +1,47 @@
+.header{
+	font-size: 46px;
+	width: 100%;
+	height: auto;
+}
+.navigation {
+font-size: 10px;
+width:25%;
+float: left;
+}
+.contain{
+	width: 100%;
+	height: 80vh;
```

Comment:
Why do you decided to use `vh`? Also why here you use this intead of `px`? Since you use that in several other cases